docs/site-example/site.conf

```diff
@@ -181,13 +181,18 @@
 
   -- Skip setup mode (config mode) on first boot
   -- setup_mode = {
-  --  skip = true,
+  --   skip = true,
```

Comment:
should I revert this correction then? or just leave it, as minor, uninteresting side-fix ;-)